tests/test_qa_t1_extraction_dual_source_1353.py

```
@@ -644,5 +586,763 @@ def test_deleted_surface_no_healed_drain_retry_residue():
+        assert retired not in web_src, f"retired machinery still present: {retired}"
+    prod = (root / "ming_sim/audience_extraction.py").read_text(encoding="utf-8")
+    assert 'code="endorsement_batch_contended"' not in prod
+    for retired_ae in (
+        "join_pending_turn_extractions",
+        "has_inflight_turn_extractions",
+        "_join_single_flight",
+        "DEFAULT_EXTRACT_JOIN_S",
+    ):
+        assert retired_ae not in prod, f"retired extraction join residue: {retired_ae}"
+    night_src = (root / "ming_sim/audience_night.py").read_text(encoding="utf-8")
+    # K10a：wait_in_flight 不按 elapsed 伪造失败；错误包 kind 仍可被 provider 终态路径使用
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)